src/RestControllers/AppointmentRestController.php

```diff
@@ -21,15 +21,50 @@
 class AppointmentRestController
 {
     private $appointmentService;
-
+    private const UNKNOW_FILTER_TYPE = "Unknow";
+    private const ARRAY_MODE = 2;
+    private const SUPPORTED_FILTER_TYPES = array(
+        "puuid",
+        "pc_title",
+        "pc_eventDate",
+        "pc_apptstatus",
+        "date_range"
+    );
+    
     public function __construct()
     {
         $this->appointmentService = new AppointmentService();
     }
 
-    public function getOne($eid)
+    /**
+     * Retrieves a single appointment by its UUID.
+     *
+     * @param string $euuid The UUID of the appointment to retrieve.
+     * @return mixed Returns the result of the appointment retrieval operation.
+     */
+    public function getOne(string $euuid)
```

Comment:
- if is possible add the return type:
public function getOne(string $euuid): TypeToReturn.

- What that means $euuid? If is possible create a more descriptive name.